gulpfile.babel.js

```diff
@@ -23,6 +23,7 @@ const nodeResources = (() => {
   return _.chain(result.toString().trim().split(/\r?\n/))
     // remove app root path
     .tap((r) => r.shift())
+    .filter((dep) => dep.match(/node_modules/g).length == 1)
```

Comment:
The entire dependency tree is still being copied to the build directory because the depth is handled by the source glob, `/**/*`. However, it is now completed in about half the time.